.gitignore

```diff
@@ -22,5 +22,7 @@ coverage
 
 # Compiled Dirs (http://nodejs.org/api/addons.html)
 build/
-dist/
+public/
```

Comment:
I'm a bit confused here, tbh. You're ignoring the public dir? But yet there are files showing from it in this commit as well. Like the index.html file - which of course, you would want to commit. So, I'm not sure what's going on here.

Reply:
Yeah, that is weird. Originally I was going to ignore the entire public dir but now that I think about it I'm just supposed to ignore bundle.js. I changed that in my .gitignore and it's still not working. I'll have to look into that.